src/main/java/com/redhat/coolstore/utils/Producers.java

```diff
@@ -1,17 +1,20 @@
 package com.redhat.coolstore.utils;
 
-import javax.enterprise.inject.Produces;
-import javax.enterprise.inject.spi.InjectionPoint;
+import jakarta.inject.Named;
+import jakarta.enterprise.inject.spi.InjectionPoint;
 import java.util.logging.Logger;
 
-
 public class Producers {
 
-    Logger log = Logger.getLogger(Producers.class.getName());
+    private Logger log = Logger.getLogger(Producers.class.getName());
+
+    @Named
+    public Logger getLog() {
+        return log;
+    }
```

Comment:
Not sure why it may have done this... Any ideas @christophermay07 ? I don't think we have a rule for this? Will have to check the analysis